crates/re_viewer/Cargo.toml

```diff
@@ -101,7 +101,7 @@ serde = { version = "1", features = ["derive"] }
 slotmap = { version = "1.0.6", features = ["serde"] }
 smallvec = { workspace = true, features = ["serde"] }
 thiserror.workspace = true
-time = { workspace = true, default-features = false, features = ["formatting"] }
```

Comment:
Is it an error to set `fault-features = false` both here and in the root `Cargo.toml`?

Reply:
It depends on the context...

If you're within the workspace, then the root `Cargo.toml` "wins" and this effectively does nothing since `time` is considered to not have any default features.

But what happens when you import `re_viewer` directly in an external project: surely the root manifest doesn't take precedence then...? or does it??! I'll test this for #2024...

Reply:
https://github.com/rerun-io/rerun/issues/2024#issuecomment-1531744702